script.js

```diff
@@ -1,4 +1,109 @@
+// Global States
+var playerTurn = 1; // 1 = Player 1; 2 = Player 2
+var gamePhase = 1; // 1 = Dice Roll; 2 = Choose Order; 3 = Conclusion
+var playerRolls = [];
+// var player1Rolls = [];
+// var player2Rolls = [];
+var playerChoice = 0; // If 1, choose 1st numerical; If 2, choose 2nd numerical
+var finalNumber1 = 0; // Stores final number for Player 1
+var finalNumber2 = 0; // Stores final number for Player 2
+
+// Function: Returns random number from 1 to 6 -inclusive
```

Comment:
definitely a good use of comments to describe the use case of each function well done!